docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -145,33 +145,30 @@ Let's review what's happening here:
   - An environment variable, or
   - Provided directly in the configuration
 
-As storing secrets in configuration is bad practice, we'll opt for using an environment variable. In this code, we're configuring the resource and supplying it to assets in the repository:
+As storing secrets in configuration is bad practice, we'll opt for using an environment variable. In this code, we're configuring the resource supplying it to our assets:
 
-```python
-## repository.py
+```python file=/guides/dagster/using_environment_variables_and_secrets/repository.py startafter=start endbefore=end
+# __init__.py
 
-from dagster import (
-    load_assets_from_package_module,
-    repository,
-    with_resources,
-)
 from my_dagster_project import assets
 from my_dagster_project.resources import github_api
 
-@repository
-def my_dagster_project():
-    return [
-        with_resources(
-            load_assets_from_package_module(assets),
-            {"github_api": github_api.configured({"access_token": {"env": "GITHUB_ACCESS_TOKEN"}})},
-        ),
-    ]
+from dagster import Definitions, load_assets_from_package_module
+
+defs = Definitions(
+    assets=load_assets_from_package_module(assets),
+    resources={
+        "github_api": github_api.configured(
+            {"access_token": {"env": "GITHUB_ACCESS_TOKEN"}}
+        )
+    },
+)
 ```
 
 Let's review what's happening here:
 
-- Using <PyObject object="with_resources" /> adds the `github_api` resource to the assets in the repository. In assets, we'll use the `github_api` resource key to reference the resource.
 - Using the <PyObject object="configured" /> method on the `github_api` resource, we can pass configuration info to the resource. In this example, we're telling Dagster to load the `access_token` from the `GITHUB_ACCESS_TOKEN` environment variable.
+- We're adding that configured resource to our `Definitions` object to that it is available for our assets.
```

Comment:
```suggestion
- We're adding that configured resource to our <PyObject object="Definitions" /> object so it's available for our assets.
```

Adding in the `PyObject` component here, so we can link to the API docs when they're ready